packages/create-osdk-app/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "create-osdk-app",
+  "private": true,
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "exports": {
+    ".": {
+      "types": "./build/types/index.d.ts",
+      "import": "./build/js/index.mjs",
+      "require": "./build/js/index.cjs"
+    },
+    "./*": {
+      "types": "./build/types/public/*.d.ts",
+      "import": "./build/js/public/*.mjs",
+      "require": "./build/js/public/*.cjs"
+    }
+  },
+  "scripts": {
+    "clean": "rm -rf lib dist types build tsconfig.tsbuildinfo",
+    "dev:transpile": "tsup --watch",
+    "fix-lint": "eslint . --fix && dprint fmt --config $(find-up dprint.json)",
+    "lint": "eslint . && dprint check  --config $(find-up dprint.json)",
+    "test": "vitest run",
+    "test:watch": "vitest",
+    "transpile": "tsup",
+    "transpileWatch": "tsup --watch",
+    "typecheck": "tsc-absolute --build",
+    "update-snapshots": "UPDATE_SNAPSHOTS=true vitest run"
```

Comment:
This is the only custom script in the package all others are the same as monorepolint